package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "temptifly",
-  "version": "0.5.7",
+  "version": "0.5.8",
```

Comment:
Version in `package.json` and `package-lock.json` should match. I normally use `npm version patch` to increment the version. CC @jeswanke - looks like perhaps you forgot to commit the `package-lock.json` change last time.